src/chat.rs

```diff
@@ -3948,16 +4022,15 @@ pub(crate) async fn add_contact_to_chat_ex(
     // this also makes sure, no contacts are added to special or normal chats
     let mut chat = Chat::load_from_db(context, chat_id).await?;
     ensure!(
-        chat.typ == Chattype::Group || chat.typ == Chattype::OutBroadcast,
-        "{} is not a group/broadcast where one can add members",
+        chat.typ == Chattype::Group || (from_handshake && chat.typ == Chattype::OutBroadcast),
+        "{} is not a group where one can add members",
         chat_id
     );
     ensure!(
         Contact::real_exists_by_id(context, contact_id).await? || contact_id == ContactId::SELF,
         "invalid contact_id {} for adding to group",
         contact_id
     );
-    ensure!(!chat.is_mailing_list(), "Mailing lists can't be changed");
```

Comment:
I removed this line because above we're ckecking `ensure!(chat.typ == Chattype::Group || chat.typ == Chattype::OutBroadcast [...])`, so, this was dead code